<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFlightClass</class>
 <widget class="QWidget" name="AddFlightClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AddFlight</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>始发地</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>131</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目的地</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>距离</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>票价</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>票量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LiEd_Start">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LiEd_End">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LiEd_Length">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LiEd_Price">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LiEd_Ticket">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="PsBt_Add">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
